cartridges/openshift-origin-cartridge-python/usr/versions/shared/bin/control

```diff
@@ -183,8 +183,15 @@ function build() {
     fi
 
     if [ -f ${OPENSHIFT_REPO_DIR}/setup.py ]; then
-        echo "Running setup.py script.."
-        ( cd $OPENSHIFT_REPO_DIR; python ${OPENSHIFT_REPO_DIR}/setup.py develop $OPENSHIFT_PYTHON_MIRROR )
+        msg=$(grep 'pandas' ${OPENSHIFT_REPO_DIR}/setup.py)
+        if [ -z "$msg" ]
+        then
+          echo "Running setup.py script.."
+          ( cd $OPENSHIFT_REPO_DIR; python ${OPENSHIFT_REPO_DIR}/setup.py develop $OPENSHIFT_PYTHON_MIRROR )
+        else
+          echo "Running pip install.."
+          ( cd $OPENSHIFT_REPO_DIR; pip install -e . )
```

Comment:
You probably want to pass `$OPENSHIFT_PYTHON_MIRROR $OPENSHIFT_PIP_TRUSTED_HOST` like we do that [a couple lines before](https://github.com/openshift/origin-server/pull/6292/files#diff-b02b0980d560b2675bbee4f4748a60e6L182).